src/lib/renderComponent.tsx

```diff
@@ -78,7 +87,7 @@ const renderComponent = <P extends {}>(
 
         // Resolve variables for this component, allow props.variables to override
         const resolvedVariables: ComponentVariablesObject = mergeComponentVariables(
-          componentVariables[displayName],
+          componentVariables[useVariablesFrom],
```

Comment:
would it still work for other components with this change?

Reply:
```
#70    useVariablesFrom = displayName,
```

Reply:
Totally agree. I will remove child API for MenuItem (for now), it is already broken anyway, and pass computed variables to `MenuItem` in `Menu.renderComponent()`